File: zad3/app/src/main/java/com/example/zad_2/ui/page3/ThirdViewModel.java
package com.example.zad_2.ui.page3;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ThirdViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public ThirdViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is third page");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: zad4/app/src/main/java/com/example/zad4/ui/page3/ThirdViewModel.kt
package com.example.zad4.ui.page3

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ThirdViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
        mText.value = "This is third page"
    }

    val text: LiveData<String>
        get() = mText
}